book/website/communication/os-comms/os-comms-issue-tracking.md

```diff
@@ -1,22 +1,25 @@
+(os-comms-issue-tracking)=
 # Issue Tracking
 
-Most software development projects have some kind of issue board to easily track the current issues in the project, such as bug fixing, rolling out new features, or community engagement plans.
+Most software development projects have some kind of issue boards to easily track current issues in the project, such as bug fixing, rolling out new features, or community engagement plans.
```

Comment:
I would keep "boards" in the singular here, but otherwise the change looks good.